integration_tests/suite/test_cdr.py

```diff
@@ -1068,6 +1076,42 @@ def test_given_call_logs_when_list_cdr_with_from_id_then_list_matching_cdr(self)
             ),
         )
 
+    @call_log(
+        **{'id': 1},
+        recordings=[{'path': '/tmp/foobar.wav'}],
+    )
+    @call_log(**{'id': 2})
+    @call_log(
+        **{'id': 3},
+        recordings=[{'path': '/tmp/foobar2.wav'}],
+    )
+    def test_given_call_logs_when_list_cdr_recorded_filter_then_list_recorded(self):
+        results = self.call_logd.cdr.list(recorded=True)
+        assert_that(
+            results,
+            has_entries(
+                filtered=2,
+                total=3,
+                items=contains_inanyorder(
+                    has_entries(id=1),
+                    has_entries(id=3),
+                ),
+            ),
+        )
+
+    @call_log(**{'id': 1})
+    @call_log(**{'id': 2})
+    def test_given_call_logs_when_list_cdr_recorded_filter_but_no_calls_recorded(self):
+        results = self.call_logd.cdr.list(recorded=True)
+        assert_that(
+            results,
+            has_entries(
+                filtered=0,
+                total=2,
+                items=empty(),
+            ),
+        )
+
```

Comment:
No objection to keep it
IMO: this test is already done in the previous test, any other corner case should be done in more closer test (dao)